lms/djangoapps/courseware/module_render.py

```diff
@@ -579,7 +579,7 @@ def rebind_noauth_module_to_user(module, real_user):
             block_wrappers.append(partial(add_staff_markup, user, has_instructor_access))
 
     # Add button for in-line analytics answer distribution
-    if getattr(settings, 'ANALYTICS_ANSWER_DIST_URL'):
+    if getattr(settings, 'ANALYTICS_DATA_URL'):
```

Comment:
Yes, PR #173 removes 'ANALYTICS_ANSWER_DIST_URL'. 'ANALYTICS_DATA_URL' was already there for Insights.